contracts/PangolinExchange.sol

```diff
@@ -155,7 +155,9 @@ contract PangolinExchange is Ownable, IAssetsExchange, ReentrancyGuardUpgradeabl
 
 
   /* ========== RECEIVE AVAX FUNCTION ========== */
-  receive() external payable {  }
+  receive() external payable {
+    emit
```

Comment:
?

Reply:
Oh, I read a comment in the audit PR about emitting an event here and started implementing it but then thought that it's not the best idea because of the reasons I stated in that discussion and I apparently forgot to remove this.